test/integration/acceptance/custom/hello_policy/main_test.go

```diff
@@ -482,6 +482,9 @@ func TestPolicies(t *testing.T) {
 						removeCrd(t, context)
 						removeClusterRole(t, context)
 					}()
+					if !base.MultipleClusters() {
+						break
+					}
 				}
```

Comment:
A more complex Multi-cluster setup has been created; each test can declare whether it works or not on a multi-cluster environment, and the secondary cluster will be configured for those tests that do.